src/baldr/turn.cc

```diff
@@ -1,12 +1,36 @@
 #include "baldr/turn.h"
 
+#include <array>
 #include <stdexcept>
 #include <string>
 #include <unordered_map>
-
 namespace valhalla {
 namespace baldr {
 
+namespace {
+constexpr std::array<Turn::Type, 360> make_turn_type_lut() {
+  std::array<Turn::Type, 360> t{};
+
+  auto fill = [&](int from_angle, int to_angle, Turn::Type type) {
+    for (int angle = from_angle; angle <= to_angle; ++angle)
+      t[static_cast<size_t>(angle)] = type;
+  };
```

Comment:
nit: It would probably a bit simpler just to have `size_t` types instead of `int`
```suggestion
  auto fill = [&](size_t from_angle, size_t to_angle, Turn::Type type) {
    for (auto angle = from_angle; angle <= to_angle; ++angle)
      t[angle] = type;
  };
```